### PR TITLE
JBIDE-20524: Fixing XULRunner crash for Mars 4.5.0 when mozila profile path does not exist. Adding missing copyrights headers

### PR DESCRIPTION
**Pull request by ibuziuk**

### CODE REVIEW

**Comment by ibuziuk** [comment]
@dgolovin, @alexeykazakov Could you please review. Tested on Windows 7 & Fedora 22

**Comment by ibuziuk** [comment]
@alexeykazakov PR was updated. I do believe now this check is save - the only two potential exceptions are: NPE (can't happen by definition) and SecurityException (catched)

File: plugins/org.jboss.tools.xulrunner.initializer/src/org/eclipse/swt/browser/BrowserInitializer.java [hunk 1]
```diff
@@ -42,6 +52,13 @@ private static boolean isGTK3Env() {
 	}
 	
 	static {
+		try {
+			createMozillaProfileDirIfNotExist();
+		} catch (SecurityException e) {
+			System.out.println(Messages.XULRunnerInitializer_Cannot_check_mozilla_profile_availability);
```
Comment (alexeykazakov) [inline plugins/org.jboss.tools.xulrunner.initializer/src/org/eclipse/swt/browser/BrowserInitializer.java]: Why not to log it via Eclipse?

File: plugins/org.jboss.tools.xulrunner.initializer/src/org/eclipse/swt/browser/PlatformUtil.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.eclipse.swt.browser;
+
+/**
+ * @author Ilya Buziuk (ibuziuk)
+ */
+public final class PlatformUtil {
```
Comment (alexeykazakov) [inline plugins/org.jboss.tools.xulrunner.initializer/src/org/eclipse/swt/browser/PlatformUtil.java]: I have already seen a few utils with for similar purposes. Can't we reuse some already existing one or create it somewhere where we can reuse it across JBT?

File: plugins/org.jboss.tools.xulrunner.initializer/src/org/eclipse/swt/browser/messages.properties [hunk 3]
```diff
@@ -1,3 +1,4 @@
 XULRunnerInitializer_Bundle_doesnt_contain=Bundle {0} doesn''t contain {1}.
 XULRunnerInitializer_Bundle_is_not_found=Bundle {0} is not found.
 XULRunnerInitializer_Cannot_get_path_to_XULRunner_from_bundle=Cannot get path to XULRunner from bundle {0}. 
+XULRunnerInitializer_Cannot_check_mozilla_profile_availability=Cannot check Mozilla profile availability
```
Comment (alexeykazakov) [inline plugins/org.jboss.tools.xulrunner.initializer/src/org/eclipse/swt/browser/messages.properties]: Let's add the path we are trying to check to this message too.

**Comment by ibuziuk** [comment]
< Why not to log it via Eclipse?
Literally there is no Activator and eclipse logger facilities in this plugin cause it is a  OSGi Fragment. From the very beginning logs were done using system "standard" output stream

< I have already seen a few utils with for similar purposes. Can't we reuse some already existing one or create it somewhere where we can reuse it across JBT. 
We cannot reuse the code from our bundles cause it will end up with cycle in the build path. There were already some GTK API which is duplicated from common because of it. However, I have contributed this OS API to common and refactor bower:
- https://github.com/jbosstools/jbosstools-base/pull/444
- https://github.com/jbosstools/jbosstools-jst/pull/519

< Let's add the path we are trying to check to this message too
done

**Comment by alexeykazakov** [comment]
LGTM
+1

**Comment by ibuziuk** [comment]
pushed to master

**Pull request merged**